### ISSUE TITLE
Office 2007 Enterprise Support

### ISSUE THREAD

**Issue by whatdoineed2do** [UMSKT/UMSKT#71]
MS office 2k7 enterprise requires activation *accepting digits* only.  Generation as per docs provides alphanumeric response
```
$ umskt -m OFFICE2K7 -p 89388-864-3870334-65364
CTTJF-WT4WX-MY8RP-CFPQJ-PKG9G
```

**Comment by CONIGUERO** [comment]
You didn't specify the Installation ID, which is a mandatory parameter for activation. Try again but oroviding the IID shown in the phone activation screen as the last argument wigh `-i`

**Comment by whatdoineed2do** [comment]
Perfect, this worked.

Would we expect `-i` to be mandatory (for all requests)?  if so I can put a PR to enforce this to prevent user error like mine above.

I see running `umskt -m OFFICE2K7 -p` causes a core dump so I can fix that along with fix above.

**Comment by CONIGUERO** [comment]
> Perfect, this worked.
> 
> Would we expect `-i` to be mandatory (for all requests)? if so I can put a PR to enforce this to prevent user error like mine above.
> 
> I see running `umskt -m OFFICE2K7 -p` causes a core dump so I can fix that along with fix above.

Glad it worked!

Yes, a PR with those improvements and bug fixes would be nice, thank you in advance for the contributions!

**Comment by whatdoineed2do** [comment]
closed by #72
